add TDMRep samples.

datalogics-alec opened the pull request.

datalogics-alec committed:
add TDMRep samples.

datalogics-pgallot commented on TextExtractTDMAware/TextExtract.csproj:
```diff
@@ -0,0 +1,14 @@
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Adobe.PDF.Library.LM.NET" Version="18.47.0" />
```
one tweak would be to replace 18.47.0 with 18.* so that this can work with current/future versions.

datalogics-pgallot commented on AddTDMProperties/AddTDMProperties.csproj:
```diff
@@ -0,0 +1,14 @@
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Adobe.PDF.Library.LM.NET" Version="18.47.0" />
```
consider replacing "18.47.0" with "18.*"

datalogics-alec committed:
update TDM sample csproj files.

at pgallot's recommendation--change the NuGet package reference to "18.*" to avoid future version conflicts.

datalogics-pgallot commented on AddTDMProperties/AddTDMProperties.cs:
```diff
@@ -0,0 +1,43 @@
+
+                String sInput = Library.ResourceDirectory + "Sample_Input/pdf_intro.pdf";
+                String sOutput = Library.ResourceDirectory + "Sample_Input/TDMRep-out.pdf";
+                String policyURI = Library.ResourceDirectory + "https://www.datalogics.com/tdm_policy_goes_here";
```
I don't think you need Library.ResourceDirectory here for policyURI.

datalogics-alec committed:
Update AddTDMProperties.cs

datalogics-alec committed:
Adjust URI to point to DL example policy.

datalogics-pgallot reviewed (approved):
Looks good to me.
